test/units/modules/network/ovs/test_openvswitch_bridge.py

```diff
@@ -20,11 +20,25 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-from units.compat.mock import patch
 from ansible.modules.network.ovs import openvswitch_bridge
+from units.compat.mock import patch, MagicMock
 from units.modules.utils import set_module_args
 from .ovs_module import TestOpenVSwitchModule, load_fixture
 
+import pytest
+
+
+@pytest.fixture
+def patched_openvswitch_bridge(monkeypatch):
+    with monkeypatch.context() as mp_ctx:
+        mocked_bridge = MagicMock()
+        mocked_bridge.return_value = {'bridge': 'test-br2', 'parent': 'test-br',
+                                      'vlan': 200, 'fail_mode': None,
+                                      'external_ids': None, 'set': None}
+        mp_ctx.setattr(openvswitch_bridge, 'map_config_to_obj', mocked_bridge)
```

Comment:
Drop `with` block and patch things directly.
```suggestion
        monkeypatch.setattr(openvswitch_bridge, 'map_config_to_obj', mocked_bridge)
```